src/site/content/en/fast/chrome-ux-report-api/index.md

```diff
@@ -0,0 +1,394 @@
+
+This API has been built from the ground up with the goals of providing developers with simple, fast, and comprehensive access to CrUX data. The CrUX API only reports _field-based_ user experience data, unlike the existing [PageSpeed Insights API](https://developers.google.com/speed/docs/insights/v5/get-started), which also runs _lab-based_ Lighthouse performance audits. The CrUX API is streamlined and can quickly serve user experience data, making it ideally suited for real-time auditing applications.
+
+To ensure that developers have access to all of the metrics that matter most—[Largest Contentful Paint](https://web.dev/lcp/) (LCP), [First Input Delay](https://web.dev/fid/) (FID), and [Cumulative Layout Shift](https://web.dev/cls/) (CLS)—the CrUX API audits and monitors these [Core Web Vitals](https://web.dev/vitals/#core-web-vitals) at both the origin and URL level.
+
+So let's dive in and see how to use it!
+
+## Querying origin data
+
+Origins in the CrUX dataset encompass all underlying page-level experiences. The example below demonstrates how to query the CrUX API for an origin's user experience data using cURL on the command line.
+
+```bash/0,3
+API_KEY="[YOUR_API_KEY]"
+curl "https://chromeuxreport.googleapis.com/v1/records:queryRecord?key=$API_KEY" \
+  --header 'Content-Type: application/json' \
+  --data '{"origin": "https://web.dev"}'
+```
+
+Run this query interactively in the [CrUX API explorer](https://developers.google.com/web/tools/chrome-user-experience-report/api/reference/rest/v1/records/queryRecord?apix=true&apix_params=%7B%22resource%22%3A%7B%22origin%22%3A%22https%3A%2F%2Fwww.google.com%22%7D%7D).
+
+{% Aside %}
+Note that all API requests must provide a value for the `key` parameter, the placeholder for which is left as `YOUR_API_KEY`. Get your own private CrUX API key at the click of a button in the official [CrUX API documentation](https://developers.google.com/web/tools/chrome-user-experience-report/api/guides/getting-started#APIKey). For convenience, the interactive [CrUX API explorer](https://developers.google.com/web/tools/chrome-user-experience-report/api/reference/rest/v1/records/queryRecord?apix=true) does not require an API key.
```

Comment:
```suggestion
All API requests must provide a value for the `key` parameter—`YOUR_API_KEY` in the example above is left as a placeholder for that value. Get your own private CrUX API key at the click of a button in the official [CrUX API documentation](https://developers.google.com/web/tools/chrome-user-experience-report/api/guides/getting-started#APIKey). For convenience, the interactive [CrUX API explorer](https://developers.google.com/web/tools/chrome-user-experience-report/api/reference/rest/v1/records/queryRecord?apix=true) does not require an API key.
```

Reply:
Accepted suggestion with a change to use the shortened API key URL.